libs/langchain/langchain/tools/clickup/prompt.py

```diff
@@ -33,3 +33,11 @@
     useful when you need to get a specific folder for the user. Given the user's workspace id you want to create a request similar to the following dictionary:
     payload = {{"space_id": "90130119692"}}
     """
+    
+CLICKUP_UPDATE_TASK_PROMPT = """
+    This tool is a wrapper around clickup's API, 
+    useful when you need to update a specific attribute of a task. Given the task id, desired attribute to change and the new value you want to create a request similar to the following dictionary:
+    payload = {{"task_id": "86a0t44tq", "attribute_name": "priority", "new_value": "1"}}
```

Comment:
Will you be updating this to include a request that has all the values that you want to support with the update API?